app/src/main/java/com/myprojects/corso/User.java

```diff
@@ -1,10 +1,17 @@
-package com.example.wert;
+package com.myprojects.corso;
+
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.FirebaseDatabase;
 
 public class User {
 
     private String username;
     private String email;
     private String password;
+    private DatabaseReference mDatabase;
+    public User(){
+        mDatabase = FirebaseDatabase.getInstance().getReference();
```

Comment:
where are you using this variable?

Reply:
I forgot to delete it. I realized that I don't really need a User class, because I can get everything that I need from the database. Am I wrong about this?